tools/github/README.md

```diff
@@ -45,6 +56,38 @@ This area is focused on the management of the Azure SDK GitHub repositories, con
  # View the help for the full set of parameters.
  get-help ./Add-AzsdkProjectIssues.ps1 -full
  ```
+ 
+ ### `Sync-AzsdkLabels.ps1`
+   _Creates or updates the set of labels expected to be common across the Azure SDK repositories, ensuring that names, descriptions, and colors share the common configuration._
+   
+   ```powershell
+   # Uses the files from the `data` directory to synchronize the common labels to all centrally managed 
+   # repositories using the default delay between each repository to guard against GitHub throttling.
+   ./Sync-AzsdkLabels.ps1 
+
+   # Synchronize the common labels to the Azure SDK for .NET repository.
+   ./Sync-AzsdkLabels.ps1 -LabelsFilePath "../data/common-labels.csv" -Languages 'net' 
+   
+   # View the help for the full set of parameters.
+   get-help ./Sync-AzsdkLabels.ps1 -full
+   ```
+   
+  ### `Sync-AzsdkLabels.ps1`
```

Comment:
Snapshot? Also looks like you copied some text from above.

Reply:
Whoops.   I did indeed.   Nice catch.